core/src/player.rs

```diff
@@ -188,6 +189,7 @@ pub struct Player {
     /// This is how we support custom SWF framerates
     /// and compensate for small lags by "catching up" (up to MAX_FRAMES_PER_TICK).
     frame_accumulator: f64,
+    recent_run_frame_timings: CircularQueue<f64>,
```

Comment:
Why not `std::collections::VecDeque`?

Reply:
Then you need something like

```
self.recent_run_frame_timings.push_back(elapsed);
if self.recent_run_frame_timings.len() > 10 {
    self.recent_run_frame_timings.pop_front();
}
```
, right?
That's what I had originally, and I felt like using a data structure that "rotates on push" automatically would be a bit more elegant.
I can go back to `VecDeque` if you want ofc.

Reply:
Yea, I'd rather avoid an extra dependency in this case. Thank you!

Reply:
Done.